simuleval/evaluator/evaluator.py

```diff
@@ -233,23 +242,29 @@ def is_finished(self, instance) -> bool:
         return instance.finish_prediction
```

Comment:
Do we still need this method after removing `while not self.is_finished(instance):`?